lib/browser/guest-view-manager.ts

```diff
@@ -179,7 +195,7 @@ const attachGuest = function (event, embedderFrameId, elementInstanceId, guestIn
     delete embedderElementsMap[oldKey];
 
     // Remove guest from embedder if moving across web views
-    if (guest.viewInstanceId !== params.instanceId) {
+    if (webViewManager && guest.viewInstanceId !== params.instanceId) {
```

Comment:
We should assert that `webViewManager` exists here instead of truthy checking, we want to do `webViewManager!` and let it error if it doesn't exist, it's a theoretical impossible case.